src/publish.proj

```diff
@@ -19,8 +19,7 @@
 
   <Target Name="PublishPackages" Condition="'$(__PublishPackages)' == 'true' and ('$(OfficialPublish)' != 'true' or '$(__BuildType)' == 'Release')">
     <PropertyGroup>
-      <RelativePath Condition="'$(RelativePath)' == ''">$(__BuildType)/pkg</RelativePath>
-      <ExpectedFeedUrl>https://$(AccountName).blob.core.windows.net/$(ContainerName)/$(RelativePath)/index.json</ExpectedFeedUrl>
+      <ExpectedFeedUrl>https://$(AccountName).blob.core.windows.net/$(ContainerName)/$(RelativePath)index.json</ExpectedFeedUrl>
```

Comment:
I said this in the original PR and I will echo it here again I think this should be in the build definition and not in the msbuild file.